src/index.ts

```diff
@@ -364,8 +364,12 @@ class SOOSDASTAnalysis {
       });
       soosLogger.info(`Scan result uploaded successfully`);
 
+
+      if (data["coreUrls"]?.length) {
```

Comment:
![image](https://github.com/soos-io/soos-dast/assets/88039131/02d40103-10e2-431c-9e81-bb8834290d56)

Reply:
going to just use the core url count rather than the spider count - and leave the json element the same